resources/skins.snapwikiskin.search/App.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <wvui-typeahead-search
+    id="searchform"
+    ref="searchForm"
+    :client="getClient"
+    :domain="domain"
+    :footer-search-text="$i18n('searchsuggest-containing').text()"
+    :suggestions-label="$i18n('searchresults').text()"
+    :accesskey="searchAccessKey"
+    :title="searchTitle"
+    :placeholder="searchPlaceholder"
+    :aria-label="searchPlaceholder"
+    :initial-input-value="searchQuery"
+    :button-label="$i18n('search').text()"
+    :form-action="action"
+    :search-language="language"
+    :show-thumbnail="showThumbnail"
+    :show-description="showDescription"
+  >
+    <input type="hidden" name="title" value="Special:Search" />
+  </wvui-typeahead-search>
+</template>
+
+<script>
+/* global SubmitEvent */
+var wvui = require("wvui");
+module.exports = {
+  name: "App",
+  components: wvui,
+  mounted: function () {
+    // access the element associated with the wvui-typeahead-search component
+    // eslint-disable-next-line no-jquery/variable-pattern
+    var wvuiSearchForm = this.$refs.searchForm.$el;
+    if (this.autofocusInput) {
+      // TODO: The wvui-typeahead-search component accepts an id prop but does not
+      // display that value as an HTML attribute on the form element.
+      wvuiSearchForm.querySelector("form").setAttribute("id", "searchform");
+      // TODO: The wvui-typeahead-search component does not accept an autofocus parameter
+      // or directive. This can be removed when its does.
+      wvuiSearchForm.querySelector("input").focus();
+    }
+  },
+  computed: {
+    /**
+     * Allow wikis eg. Hebrew Wikipedia to replace the default search API client
```

Comment:
We are not the WMF.

Reply:
Yes, but this will be deployed on Miraheze and is good to have

Reply:
I was talking about the mention of Hebrew Wikipedia in the comment, not the code. :)